File: lang/php/src/muse/algorithms/number_theory/Coprimality.php
<?php
    namespace muse\algorithms\number_theory\Coprimality;

    require_once "muse/algorithms/number_theory/GreatestCommonDivisor.php";

    use muse\algorithms\number_theory\GreatestCommonDivisor;

    function reduceToBinaryGCD(int $m, int $n): bool {
        return GreatestCommonDivisor\iterativeBinaryGCD($m, $n) == 1;
    }

    function reduceToEuclidean(int $m, int $n): bool {
        return GreatestCommonDivisor\iterativeEuclidean($m, $n) == 1;
    }
?>

File: go/muse/algorithms/number_theory/coprimality/coprimality.go
package coprimality

import "muse/algorithms/number_theory/greatest_common_divisor"

func ReduceToBinaryGCD(m, n int64) bool {
	return greatest_common_divisor.IterativeBinaryGCD(m, n) == 1
}

func ReduceToEuclidean(m, n int64) bool {
	return greatest_common_divisor.IterativeEuclidean(m, n) == 1
}
